src/AmalgamatorSession.ts

```diff
@@ -426,6 +426,23 @@ export class AmalgamatorSession extends LoggingDebugSession {
         response.body = variables.body;
         this.sendResponse(response);
     }
+    
+    protected async evaluateRequest(response: DebugProtocol.EvaluateResponse, args: DebugProtocol.EvaluateArguments): Promise<void> {
+        const [childDap, childFrameId] = this.frameHandles.get(args.frameId);
```

Comment:
Because frameid can be undefined, you can get undefined return from get. Please simply add a check in that case and return the appropriate error response to the client.

If one day we need to support frameid being undefined we can add that, but in the meantime let's not leave an unhandled exception in the code.